charts/galoy/templates/admin-panel-ingress.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if (index .Values "admin-panel").ingress.enabled }}
+apiVersion: {{ include "common.capabilities.ingress.apiVersion" . }}
```

Comment:
Don't think this should be overridable. A change to the `apiVersion` would be a breaking change also for this template structure. Better to hard code it here to know what schema we need to apply.

Reply:
Fixed